### PR TITLE
Create Main-de.strings

### PR DESCRIPTION
**Pull request by Luca1719**
Localized by Luca (for reference: Reddit: u/Luca1719)

### CODE REVIEW

File: Main-de.string [hunk 1]
```diff
@@ -0,0 +1,717 @@
...
+"ypj-kv-Bcb.title" = "Keine";
+
+/* Class = "NSMenuItem"; title = "Show Substitutions"; ObjectID = "z6F-FW-3nz"; */
+"z6F-FW-3nz.title" = "Ersetzungen anzeigen";
+
+/* Class = "NSTextFieldCell"; title = "Line Weight"; ObjectID = "zxt-da-umM"; */
+"zxt-da-umM.title" = "Linienbreite";
+
+/* Class = "NSTextFieldCell"; title = "Decrease Line Weight"; ObjectID = "rij-ew-yIo"; */
+"rij-ew-yIo.title" = "Linienbreite verringern";
```
Comment (zeisss) [inline Main-de.string:705]: Here you translated the same string (Line weight) differently. I think "Liniendicke" is the better word.

**Comment by rampatra** [comment]
Thanks a lot @Luca1719 and @zeisss for the help.

Just one minor thing before I merge this. The file name should be `Main.strings` and `Localizable.strings` and should be inside `de.lproj` folder.

**Comment by rampatra** [comment]
I am thinking it would be better for me to reorganize the files in your fork and then merge them. Hope that's okay.

File: Main-de.string [hunk 2]
```diff
@@ -0,0 +1,717 @@
...
+"GUa-eO-cwY.title" = "Standard benutzen";
+
+/* Class = "NSMenuItem"; title = "Font"; ObjectID = "Gi5-1S-RQB"; */
+"Gi5-1S-RQB.title" = "Schriftart";
+
+/* Class = "NSMenuItem"; title = "Writing Direction"; ObjectID = "H1b-Si-o9J"; */
+"H1b-Si-o9J.title" = "Schriftrichtung";
+
+/* Class = "NSTextFieldCell"; title = "Hold down Option Key while drawing Rectangles or Circles to fill a semi-transparent color."; ObjectID = "H6d-jK-d3o"; */
+"H6d-jK-d3o.title" = "Halte die Option-Taste während du Rechtecke und Kreise malst, um sie mit einer halbtransparenten Farbe zu füllen.";
```
Comment (Suplanus) [inline Main-de.string:159]: Better:
"Halte die Option-Taste während Du Rechtecke oder Kreise malst, um sie mit einer halbtransparenten Farbe zu füllen."

File: Main-de.string [hunk 3]
```diff
@@ -0,0 +1,717 @@
...
+"HlN-Sj-S7B.ibShadowedToolTip" = "Alles löschen";
+
+/* Class = "NSMenu"; title = "View"; ObjectID = "HyV-fh-RgO"; */
+"HyV-fh-RgO.title" = "Ansicht";
+
+/* Class = "NSMenuItem"; title = "Subscript"; ObjectID = "I0S-gh-46l"; */
+"I0S-gh-46l.title" = "Tiefsatz";
+
+/* Class = "NSMenuItem"; title = "Open…"; ObjectID = "IAo-SY-fd9"; */
+"IAo-SY-fd9.title" = "Öffne ...";
```
Comment (Suplanus) [inline Main-de.string:180]: Better:
"Öffnen..."

File: Main-de.string [hunk 4]
```diff
@@ -0,0 +1,717 @@
...
+
+/* Class = "NSTextFieldCell"; title = "Highlight Color"; ObjectID = "IFr-N7-z9M"; */
+"IFr-N7-z9M.title" = "Markierfarbe";
+
+/* Class = "NSMenuItem"; title = "Justify"; ObjectID = "J5U-5w-g23"; */
+"J5U-5w-g23.title" = "Justieren";
```
Comment (Suplanus) [inline Main-de.string:186]: Better:
"Ausrichten"

File: Main-de.string [hunk 5]
```diff
@@ -0,0 +1,717 @@
...
+
+/* Class = "NSMenuItem"; title = "Use None"; ObjectID = "J7y-lM-qPV"; */
+"J7y-lM-qPV.title" = "Keine benutzen";
+
+/* Class = "NSButtonCell"; title = "C"; ObjectID = "JM6-OU-IuG"; */
+"JM6-OU-IuG.title" = "C";
+
+/* Class = "NSButton"; ibShadowedToolTip = "Move Annotate Controls"; ObjectID = "JvL-1K-hfp"; */
+"JvL-1K-hfp.ibShadowedToolTip" = "Kommentarkontrollen bearbeiten";
```
Comment (Suplanus) [inline Main-de.string:195]: Better:
"Markier-Steuerelemente verschieben"

File: Main-de.string [hunk 6]
```diff
@@ -0,0 +1,717 @@
...
+"Kd2-mp-pUS.title" = "Alles zeigen";
+
+/* Class = "NSButton"; ibShadowedToolTip = "Erase all"; ObjectID = "KlH-Uf-oc1"; */
+"KlH-Uf-oc1.ibShadowedToolTip" = "Alles löschen";
+
+/* Class = "NSTextFieldCell"; title = "Undo"; ObjectID = "KvF-sv-PWx"; */
+"KvF-sv-PWx.title" = "Rückgängig";
+
+/* Class = "NSMenuItem"; title = "Bring All to Front"; ObjectID = "LE2-aR-0XJ"; */
+"LE2-aR-0XJ.title" = "Alle nach vorne bringen";
```
Comment (Suplanus) [inline Main-de.string:210]: Better:
"Alle nach vorne bringen"

File: Main-de.string [hunk 7]
```diff
@@ -0,0 +1,717 @@
...
+"Q5e-8K-NDq.title" = "Schriftarten anzeigen";
+
+/* Class = "NSMenuItem"; title = "Zoom"; ObjectID = "R4o-n2-Eq4"; */
+"R4o-n2-Eq4.title" = "Zoom";
+
+/* Class = "NSMenuItem"; title = "\tRight to Left"; ObjectID = "RB4-Sm-HuC"; */
+"RB4-Sm-HuC.title" = "\tVon rechts zu links";
+
+/* Class = "NSTextFieldCell"; title = "Set the weight of your annotations"; ObjectID = "RIq-pW-x2s"; */
+"RIq-pW-x2s.title" = "Setze das Schriftgewicht deiner Kommentare";
```
Comment (Suplanus) [inline Main-de.string:291]: Better:
"Setze die breite der Markierungen"

File: Main-de.string [hunk 8]
```diff
@@ -0,0 +1,717 @@
...
+
+/* Class = "NSMenuItem"; title = "Superscript"; ObjectID = "Rqc-34-cIF"; */
+"Rqc-34-cIF.title" = "Hochgestellt";
+
+/* Class = "NSMenuItem"; title = "Select All"; ObjectID = "Ruw-6m-B2m"; */
+"Ruw-6m-B2m.title" = "Alles auswählen";
+
+/* Class = "NSMenuItem"; title = "Jump to Selection"; ObjectID = "S0p-oC-mLd"; */
+"S0p-oC-mLd.title" = "Zuw Auswahl springen";
```
Comment (Suplanus) [inline Main-de.string:300]: "Zur Auswahl springen"

File: Main-de.string [hunk 9]
```diff
@@ -0,0 +1,717 @@
...
+"SRy-I9-5Ye.title" = "Alle Markierungskurzbefehle";
+
+/* Class = "NSButtonCell"; title = "C"; ObjectID = "Sye-Hc-SvQ"; */
+"Sye-Hc-SvQ.title" = "C";
+
+/* Class = "NSButtonCell"; title = "Round Rect"; ObjectID = "TLY-JS-GmD"; */
+"TLY-JS-GmD.title" = "Abgerundetes Rechteck";
+
+/* Class = "NSTextFieldCell"; title = "Favorite Colors"; ObjectID = "TOY-jq-L3N"; */
+"TOY-jq-L3N.title" = "Favoritfarbe";
```
Comment (Suplanus) [inline Main-de.string:315]: "Favorisierte Farben"

File: Main-de.string [hunk 10]
```diff
@@ -0,0 +1,717 @@
...
+"VEn-hr-usv.title" = "Abgerundetes Rechteck";
+
+/* Class = "NSMenuItem"; title = "Center"; ObjectID = "VIY-Ag-zcb"; */
+"VIY-Ag-zcb.title" = "Mitte";
+
+/* Class = "NSMenuItem"; title = "Hide Others"; ObjectID = "Vdr-fp-XzO"; */
+"Vdr-fp-XzO.title" = "Andere verstecken";
+
+/* Class = "NSMenuItem"; title = "Italic"; ObjectID = "Vjx-xi-njq"; */
+"Vjx-xi-njq.title" = "Kursic";
```
Comment (Suplanus) [inline Main-de.string:336]: "Kursiv"

File: Main-de.string [hunk 11]
```diff
@@ -0,0 +1,717 @@
...
+"Xif-nS-TLX.label" = "Mauszeiger";
+
+/* Class = "NSMenuItem"; title = "Find…"; ObjectID = "Xz5-n4-O0W"; */
+"Xz5-n4-O0W.title" = "Finden ...";
+
+/* Class = "NSButton"; ibShadowedToolTip = "Second Fav Color"; ObjectID = "Y5B-HG-lAJ"; */
+"Y5B-HG-lAJ.ibShadowedToolTip" = "Zweite favorisierte Farbe";
+
+/* Class = "NSMenuItem"; title = "Find and Replace…"; ObjectID = "YEy-JH-Tfz"; */
+"YEy-JH-Tfz.title" = "Finden und ersetzen ...";
```
Comment (Suplanus) [inline Main-de.string:369]: "Suchen und ersetzen..."

File: Main-de.string [hunk 12]
```diff
@@ -0,0 +1,717 @@
...
+"c8a-y6-VQd.title" = "Transformationen";
+
+/* Class = "NSMenuItem"; title = "Use None"; ObjectID = "cDB-IK-hbR"; */
+"cDB-IK-hbR.title" = "Keine benutzen";
+
+/* Class = "NSTextFieldCell"; title = "Set your three colors for annotating your screen"; ObjectID = "cJW-PG-OkU"; */
+"cJW-PG-OkU.title" = "Setze deine drei Farben zum Kommentieren des Bildes";
+
+/* Class = "NSTextFieldCell"; title = "Start Presentify when your mac starts"; ObjectID = "cMw-Qt-4pz"; */
+"cMw-Qt-4pz.title" = "Presentify starten, wenn du deinen Mac startest";
```
Comment (Suplanus) [inline Main-de.string:441]: "Presentify starten, wenn dein Mac startet"

File: Main-de.string [hunk 13]
```diff
@@ -0,0 +1,717 @@
...
+/* Class = "NSButtonCell"; title = "C"; ObjectID = "iGA-Un-7tt"; */
+"iGA-Un-7tt.title" = "C";
+
+/* Class = "NSTextFieldCell"; title = "Hold down Shift Key while Free Hand drawing to draw a straight line."; ObjectID = "ibz-5L-RfG"; */
+"ibz-5L-RfG.title" = "Halte die Shift-Taste während du Freihand zeichnest um eine gerade Linie zu zeichen.";
+
+/* Class = "NSMenu"; title = "Baseline"; ObjectID = "ijk-EB-dga"; */
+"ijk-EB-dga.title" = "Grundlinie";
+
+/* Class = "NSMenuItem"; title = "Kern"; ObjectID = "jBQ-r6-VK2"; */
```
Comment (Suplanus) [inline Main-de.string:533]: What means "Kern" in english?

File: Main-de.string [hunk 14]
```diff
@@ -0,0 +1,717 @@
...
+"lY5-ew-VdV.title" = "C";
+
+/* Class = "NSMenuItem"; title = "Solid"; ObjectID = "lt6-jb-Sde"; */
+"lt6-jb-Sde.title" = "Undurchlässig";
+
+/* Class = "NSMenuItem"; title = "Check Grammar With Spelling"; ObjectID = "mK6-2p-4JG"; */
+"mK6-2p-4JG.title" = "Grammatik mit Rechtschreibung prüfen";
+
+/* Class = "NSButtonCell"; title = "Highlight Cursor at app launch"; ObjectID = "mXB-PS-wv2"; */
+"mXB-PS-wv2.title" = "Beim App-Launch Mauszeiger markieren";
```
Comment (Suplanus) [inline Main-de.string:567]: "Beim Programmstart Mauszeiger markieren"

File: Main-de.string [hunk 15]
```diff
@@ -0,0 +1,717 @@
...
+"vMM-vc-8Oy.ibShadowedToolTip" = "Pfeile";
+
+/* Class = "NSMenuItem"; title = "Clear Menu"; ObjectID = "vNY-rz-j42"; */
+"vNY-rz-j42.title" = "Menü leeren";
+
+/* Class = "NSButtonCell"; title = "Highlight with a full circle instead of a ring"; ObjectID = "vbl-Bg-qj4"; */
+"vbl-Bg-qj4.title" = "Mit einem vollen Kreis anstatt einem Ring markieren";
+
+/* Class = "NSTextFieldCell"; title = "1st Fav Color"; ObjectID = "vli-ph-UGB"; */
+"vli-ph-UGB.title" = "Erste favorisierte Farve";
```
Comment (Suplanus) [inline Main-de.string:666]: "Erste favorisierte Farbe"

File: Main-de.string [hunk 16]
```diff
@@ -0,0 +1,717 @@
...
+"zxt-da-umM.title" = "Linienbreite";
+
+/* Class = "NSTextFieldCell"; title = "Decrease Line Weight"; ObjectID = "rij-ew-yIo"; */
+"rij-ew-yIo.title" = "Linienbreite verringern";
+
+/* Class = "NSTextFieldCell"; title = "Increase Line Weight"; ObjectID = "Xns-of-diF"; */
+"Xns-of-diF.title" = "Liniendicke vergrößern";
+
+/* Class = "NSTextFieldCell"; title = "Draw with Touch"; ObjectID = "6fu-rC-JLd"; */
+"6fu-rC-JLd.title" = "Mit Berührung zeichen";
```
Comment (Suplanus) [inline Main-de.string:711]: "Mit Touch zeichnen"

**Review by Suplanus — CHANGES_REQUESTED** [review]

**Comment by rampatra** [comment]
If someone else can please corroborate on the comments made by @Suplanus and @zeisss then I can make the changes.

**Comment by rampatra** [comment]
@zeisss Also, I have given a free license of Presentify to @Luca1719 and @Suplanus so if you need one then please let me know. Would be happy to share one.

Thanks a lot for your help 🙏

**Comment by zeisss** [comment]
No problem. No need for a license. Happy to help.

**Pull request merged**